hotshot-stake-table/src/utils.rs

```diff
@@ -20,7 +20,12 @@ pub trait ToFields<F: Field> {
 
 /// convert a U256 to a field element.
 pub(crate) fn u256_to_field<F: PrimeField>(v: &U256) -> F {
-    let mut bytes = vec![0u8; 32];
-    v.to_little_endian(&mut bytes);
+    let bytes: [u8; 32] = v.to_le_bytes();
     F::from_le_bytes_mod_order(&bytes)
 }
+
+#[inline]
+/// A helper function to compute the quorum threshold given a total amount of stake.
+pub fn one_honest_threshold(total_stake: U256) -> U256 {
```

Comment:
Are we supposed to use the quorum threshold fn provided by membership?

Reply:
can you point me to it? I move this from some utils.rs i remember.

Reply:
Like here: https://github.com/EspressoSystems/espresso-network/blob/861a30e83490f2f1fa8a951c5275a738a06764a8/hotshot-types/src/traits/election.rs#L126

We can also leave it as a tech debt and discuss later.

Reply:
i took a look, so `success_threshold()` here means `2/3f+1`, but here we only need a single honest signature, which is a different threshold. 

In future PRs, we could add an additional method in the `trait Membership` ? but definitely future work.